autoload/neomake/makers/ft/elixir.vim

```diff
@@ -19,3 +19,11 @@ function! neomake#makers#ft#elixir#credo()
       \ 'errorformat': '[%t] %. %f:%l:%c %m'
       \ }
 endfunction
+
+function! neomake#markers#ft#elixir#dogma()
+    return {
+      \ 'exe': 'mix',
+      \ 'args': ['dogma', '%:p', '--format=flycheck'],
+      \ 'errorformat': '%f:%l:%c: %.: %m'
```

Comment:
If all are errors, please add `%E` to the beginning of the errorformat.

Also, please add it to the README.